### DISCUSSION TITLE
Wish: pressing 'drives dropdown' button 2nd time must hide the drives listbox

### DISCUSSION THREAD

**Discussion by Alexey-T** [doublecmd/doublecmd#1082] [Suggestions]
### Description

![Screenshot from 2023-02-20 17-09-16](https://user-images.githubusercontent.com/4154978/220133522-afe6b312-db5b-4ef9-a014-7c6fa93018df.png)

I marked this 'drives button' on picture. I am on Linux x64 gtk2 version. pressing the btn 2nd time DO NOT hide the listbox below the button!

**Comment by JonnyTech** [comment]
I have tried latest Linux x64 gtk2 (and qt) build and cannot replicate the issue. Please confirm the version you are running from: Help >> About >> Copy to clipboard. Do you have any software running that may interfere with mouse clicks?

**Comment by Alexey-T** [comment]
```
Double Commander
Version: 1.0.10
Revision: 239
Commit: d00f8b7b7
Build date: 2023/01/21
Lazarus: 2.2.2.0
FPC: 3.2.2
Platform: x86_64-Linux-gtk2
OS version: Ubuntu 22.04.1 LTS

Widgetset library: GTK 2.24.33
```

**Comment by JonnyTech** [comment]
Thanks that is the same version that I tested. Can you create a new user account on you computer and try there? It may be a conflict with other running software or a (mis)configuration issue.

**Comment by avmaksimov** [comment]
> Thanks that is the same version that I tested. Can you create a new user account on you computer and try there? It may be a conflict with other running software or a (mis)configuration issue.

I can confirm this strange behavior and I think the reason is in a bug in FCL. Because sometimes OK, sometimes I can't open and sometimes I can't close. On Windows All OK.
